astropy/io/ascii/daophot.py

```diff
@@ -165,15 +195,105 @@ class DaophotData(core.BaseData):
     start_line = 0
     comment = r'\s*#'
 
+    def __init__(self):
+        core.BaseData.__init__(self)
+        self.is_multiline = False
```

Comment:
If instead `is_multiline = False` is defined just under `start_line = 0` (5 lines above), there there is no need to write an `__init__` at all. Again, 2 lines of code saved ;-)